libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/serial_api.c

```diff
@@ -207,6 +245,11 @@ void UART1_Handler()
 {
     uart_irq(UART_1, 1);
 }
+
+void UART2_Handler()
```

Comment:
I understood we have to use NVIC_SetVector() API. Is it right?
But our chip(W7500) hasn't register for vector table as VTOR(Vector Table Offset Register).